### PR TITLE
Feat:Result and Rules page done

### PR DESCRIPTION
**Pull request by tushargoyal812**
quiz-result
quiz-rules

### CODE REVIEW

File: maha-result/result.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+
+
+.option-btn{
+    width: 25rem;
+    padding: 1.5rem 8rem;
+    background-color: #cbd5e1;
+    color: black;
```
Comment (sanjay-xdr) [inline maha-result/result.css:7]: use hex or rgba rather than name

**Review by sanjay-xdr — COMMENTED** [review]
Looks good

File: maha-result/result.css (hunk 1 above)
Comment (tushargoyal812) [reply maha-result/result.css:7]: ok

**Review by syed-shaida-hussain — CHANGES_REQUESTED** [review]
just use variables for colors. Everything else looks fine

**Pull request merged**